pages/singleProfile/index.js

```diff
@@ -0,0 +1,106 @@
+import styled from "styled-components";
+
+const singleProfile = ({userArray, clickedName}) => {
+  const clickedNameObject = {name: `${clickedName}`};
+  const checkClickedUser = userArray.filter(user => {
+    return user.name == clickedNameObject.name;
+  });
+
+  return (
+    <>
+      <h6>Login/My Contacts/Contact Profile</h6>
+      {checkClickedUser.map(user => {
+        return (
+          <ContainerDiv key={`key-section-${user.id}`}>
+            <ContainerProfileImage
+              src={user.image}
+              alt={user.name}
+              height="50vh"
+              width="50vw"
+            />
+            <ContainerTextareaName disabled value={user.name} />
+            <ContainerTextareaAddress disabled value={user.address} />
+            <ContainerTextareaEmail disabled value={user.email} />
+            <ContainerTextareaPhone disabled value={user.phone} />
+            <ContainerTextareaWebsite disabled value={user.website} />
+            <ContainerButtonEdit>Edit</ContainerButtonEdit>
+          </ContainerDiv>
+        );
+      })}
+    </>
+  );
+};
+
+export default singleProfile;
+
+const ContainerDiv = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 10px;
+
+  margin-left: 10%;
+  width: 80%;
+  gap: 3px;
+
+  background: rgba(255, 255, 255, 0.1);
+  border-radius: 20px;
+  box-shadow: 0 4px 30px rgba(0, 0, 0, 0.1);
+  backdrop-filter: blur(2.2px);
+  -webkit-backdrop-filter: blur(2.2px);
+  border: 1px solid rgba(255, 255, 255, 0.5);
+`;
+const ContainerTextareaName = styled.textarea`
+  background-image: url("icon_name.svg");
+  background-repeat: no-repeat;
+  background-position: left;
+  background-size: contain;
+  padding-left: 40px;
+  border-radius: 20px;
+`;
+const ContainerTextareaAddress = styled.textarea`
+  background-image: url("icon_address.svg");
+  background-repeat: no-repeat;
+  background-position: left;
+  background-size: contain;
+  padding-left: 40px;
+  border-radius: 20px;
+`;
+const ContainerTextareaEmail = styled.textarea`
+  background-image: url("icon_email.svg");
+  background-repeat: no-repeat;
+  background-position: left;
+  background-size: contain;
+  padding-left: 40px;
+  border-radius: 20px;
+`;
+const ContainerTextareaPhone = styled.textarea`
+  background-image: url("icon_phone.svg");
+  background-repeat: no-repeat;
+  background-position: left;
+  background-size: contain;
+  padding-left: 40px;
+  border-radius: 20px;
+`;
+const ContainerTextareaWebsite = styled.textarea`
+  background-image: url("icon_website.svg");
+  background-repeat: no-repeat;
+  background-position: left;
+  background-size: contain;
+  padding-left: 40px;
```

Comment:
Please use relative units at this point.

Reply:
Fixed.